go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/aquasecurity/trivy-db v0.0.0-20231005141211-4fc651f7ac8d
 	github.com/aquasecurity/trivy-iac v0.7.1
 	github.com/aquasecurity/trivy-java-db v0.0.0-20240109071736-184bd7481d48
-	github.com/aquasecurity/trivy-kubernetes v0.5.9-0.20231203080602-50a069120091
```

Comment:
you can discard `go.mod` and `go.sum` changes, upstream already include `trivy-kubernetes v0.6.1` with you latest changes

Reply:
@chen-keinan I've addressed the comments you've made. Please revisit the changes. Thanks.